nix/README.md

```diff
@@ -1,29 +1,35 @@
 ### Nix support
 
+### Quickstart
+
+- `sudo bash -c “mkdir /nix && chmod 755 /nix && chown -R $USERNAME /nix”`
+- Run the multi-user command from `https://nixos.org/download.html#nix-install-linux`
+- Enable flakes: `mkdir ~/.config/nix; echo "experimental-features = nix-command flakes" > ~/.config/nix.conf`
```

Comment:
What are flakes? Even a once sentence description would be useful/ or a link to good documentation with the explanation.